spec/controllers/connections_controller_spec.rb

```diff
@@ -0,0 +1,80 @@
+require 'spec_helper'
+
+describe ConnectionsController do
+  it { should use_before_action :require_login }
+
+  describe 'GET index' do
+    it "sets @connections to the current user's guides (people they follow)" do
+      riggs = Fabricate :user, full_name: 'Riggs'
+      3.times { Fabricate :connection, follower: riggs }
```

Comment:
You should be able to achieve the same with `Fabricate.times(3, :connection, ...)`